src/components/PopupWithForm.js

```diff
@@ -4,14 +4,15 @@ class PopupWithForm extends Popup {
   constructor(popupSelector, handleFormSubmit) {
     super({ popupSelector });
     this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._formButton = this._popupElement.querySelector(".modal__button");
+    this._formButtonText = this._formButton.textContent;
     this._handleFormSubmit = handleFormSubmit;
     this._inputList = [...this._popupForm.querySelectorAll(".modal__input")]; //Array.from
   }
 
-  // close() {
-  //   // this._popupForm.reset();
-  //   super.close();
-  // }
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.

Reply:
* You haven't corrected one thing https://github.com/juanaima520/se_project_aroundtheus/pull/1#discussion_r1764361356